### PR TITLE
SwerveModuleController + Subsystem (with Drivetrain Constants, and a possible DrivetrainCommands)

### LINKED ISSUE #11: Convert swerve subsystem and module controller to Java
Converting the Kotlin files from 2023-robot-code to java that were the swerve subsystem, and swerve module controller. Testing and debugging not yet needed.

### PR DESCRIPTION
**Pull request by aaditsangvikar**

### CODE REVIEW

**Review by ethantheberg — APPROVED** [review]
seems approximately legit

File: src/main/java/frc/robot/constants/DrivetrainConstants.java
Comment (zanealoia) [inline src/main/java/frc/robot/constants/DrivetrainConstants.java:1]: yikes

**Review by zanealoia — APPROVED** [review]
You did a good job. In the future, more documentation is needed.

**Review by acuaroo — APPROVED** [review]
Good

**Pull request merged**